drydock/patches/uwsgi-config

```diff
@@ -12,3 +12,5 @@ no-defer-accept = true
 master = true
 py-call-osafterfork = true
 vacuum = true
+hook-master-start = unix_signal:1 gracefully_kill_them_all
+disable-logging = true
```

Comment:
How does the logs in stdout out look with all the settings applied?

Reply:
Only edx-platform stdout and stderr is available. It removes only request logs such as:

```
[pid: 9|app: 0|req: 2/3] 10.100.2.19 () {48 vars in 778 bytes} [Thu Sep  5 01:26:36 2024] GET / => generated 9508 bytes in 1955 msecs (HTTP/1.1 200) 7 headers in 663 bytes (1 switches on core 0)
```